app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -91,6 +91,10 @@ $form-feedback-valid-color: inherit;
   }
 }
 
+img {
+  width: 100%
```

Comment:
This breaks the spacing of the logo. It also seems that this is only used once in your commits. You can just make this a html style attribute.

Reply:
everything else LGTM

Reply:
We will generally want this in place, so it would be better to go fix the logo.